dask/bag/tests/test_text.py

```diff
@@ -30,7 +30,7 @@
 
 expected = "".join([files[v] for v in sorted(files)])
 
-fmt_bs = [(fmt, None) for fmt in compr] + [(None, "10 B")]  # type: ignore
+fmt_bs = [(fmt, None) for fmt in compr] + [(None, "10 B")]
```

Comment:
Wow, I've wanted a fix for this for a long time! cf https://github.com/python/mypy/issues/5492